README.md

```diff
@@ -89,6 +89,7 @@
 - [vast](https://github.com/lydiandy/vast) - A simple tool for vlang, generate v source file to AST json file.
 - [symlinker](https://github.com/serkonda7/symlinker) - A small Linux tool to manage symlinks.
 - [runner](https://github.com/Naheel-Azawy/runner) - A tool that automates running/compiling code written in various programming languages.
+- [props](https://github.com/StunxFS/props) - Property support like Java for V 
```

Comment:
Please add a `.` at the end of the sentence.

Reply:
Ready